core/common/src/test/java/alluxio/clock/FakeClock.java

```diff
@@ -14,13 +14,13 @@
 /**
  * A modifiable clock useful for testing.
  */
-public final class TestClock implements Clock {
-  private long mTime;
+public final class FakeClock implements Clock {
```

Comment:
Isn't this more of a modifiable clock or manual clock? "fake" sounds a bit strange.